src/unix/bsd/apple/mod.rs

```diff
@@ -4968,6 +4973,19 @@ pub const PROC_PIDTASKINFO: c_int = 4;
 pub const PROC_PIDTHREADINFO: c_int = 5;
 pub const PROC_PIDVNODEPATHINFO: c_int = 9;
 pub const PROC_PIDPATHINFO_MAXSIZE: c_int = 4096;
+
+pub const PROC_PIDLISTFDS: c_int = 1;
+pub const PROC_PIDLISTFD_SIZE: c_int = 8;
```

Comment:
Can this be `core::mem::size_of::<proc_fdinfo>()` to match the headers?

Reply:
Didn't realize the MSRV was updated, then definitely!